### PR TITLE
Fix: for multiple header in parameter support

### PR DESCRIPTION
**Pull request by tehcyx**
Hi,

We, a team from SAP working on [API Hub](https://api.sap.com) and [YaaS](https://api.yaas.io), started using this converter to convert our RAML definitions into Swagger. We found that some of our RAML files were failing to convert so we had to make some changes, and we wanted to share these changes with you. We collected the list of changes we made in this [CHANGELOG](https://github.com/tehcyx/raml-to-swagger/blob/master/CHANGELOG.md) and will provide each fix as a different pull request for you so that you can pick the ones that you want to support.

**The issue to fix**
Header with "collectionFormat: 'multi'" is not supported

**The fix**
Change the 'multi' to 'csv' so it accepts comma separated lists.

Best regards,
Daniel Roth - SAP Hybris

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -294,11 +294,12 @@ function parseParametersList(params, inValue) {
...
 
      if (value.repeat === true) {
-       assert(['query', 'formData'].indexOf(inValue) !== -1);
+       //changes for multiple header support - added in header to the assert check
+       assert(['query', 'formData', 'header'].indexOf(inValue) !== -1);
        srParameter = {
          type: 'array',
          items: srParameter,
-         collectionFormat: 'multi'
+         collectionFormat: 'csv' // collectionFormat: 'multi' not supported for headers, use csv instead
```
Comment (IvanGoncharov) [inline src/index.js:302]: @tehcyx You apply `csv` not only to `header` but also `query` and `formData`.
Is it intended, or I can add `inValue === 'header' ? 'csv' : 'multi'`?

File: src/index.js [hunk 2]
```diff
@@ -294,11 +294,12 @@ function parseParametersList(params, inValue) {
      }
 
      if (value.repeat === true) {
-       assert(['query', 'formData'].indexOf(inValue) !== -1);
+       //changes for multiple header support - added in header to the assert check
```
Comment (IvanGoncharov) [inline src/index.js:297]: You wrote excellent PR description 👍  no need to describe your changes in comments

File: src/index.js (hunk 1 above)
Comment (tehcyx) [reply src/index.js:302]: You can add that line, we should've actually done it that way. Our concern was the headers.

File: src/index.js (hunk 2 above)
Comment (tehcyx) [reply src/index.js:297]: Thanks, I just added the PR description after we created the changelog file from the comments we added. That's the only reason :)

**Comment by IvanGoncharov** [comment]
@tehcyx I apply this fix in https://github.com/APIs-guru/raml-to-swagger/commit/4c095b40bd48ea36402325828a6b4a954cf5e129 
Thank you 👍

**Pull request closed without merging**